### ISSUE TITLE
unnecessary code generation

### ISSUE THREAD

**Issue by icas711** [Kotelnikovekb/extension_theme_generator#1]
The $AppThemeColors class is generated in the "*.g.dart" files marked by annotation "@freezed"

**Comment by icas711** [comment]
Во всех файлах "*.g.dart" проекта генерирует class $AppThemeColors.
Если добавить в 36 строку "_allFields.clear();" в color_generator.dart. Нормально работает.
Спасибо удобный плагин, значительно ускоряет

**Comment by Kotelnikovekb** [comment]
Да, спасибо большое за подсказку. я тоже столкнулся с этой проблемой. Сейчас я добавил этот момент.

> Во всех файлах "*.g.dart" проекта генерирует class $AppThemeColors. Если добавить в 36 строку "_allFields.clear();" в color_generator.dart. Нормально работает. Спасибо удобный плагин, значительно ускоряет
